templates/store/store.html

```diff
@@ -36,7 +52,15 @@ <h3>Juju solutions for <br class="u-hide--small"/>big data</h3>
     <div class="col-6">
       <div class="row">
         <div class="col-2 u-align--center">
-          <img src="https://assets.ubuntu.com/v1/3fe6b167-CHARM-FILEBEATS.svg" alt="">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/3fe6b167-CHARM-FILEBEATS.svg",
+              alt="",
```

Comment:
Missing alt

Reply:
As was the previous because this image is largely presentational and therefore should be given a blank alt. See https://webaim.org/techniques/alttext/